compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/tree/STActionNode.java

```diff
@@ -1,7 +1,7 @@
 /*
- *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *  Copyright (c) 2020, WSO2 LLC. (http://www.wso2.com).
```

Comment:
```suggestion
 *  Copyright (c) 2020, WSO2 LLC. (http://www.wso2.com).
```
@hasithaa since we are changing `Inc` to `LLC` should we change the year of these files to `2023`?

Reply:
If we're changing so we need to update the metadata field name from `createdYear` to a more appropriate name

Reply:
It should be 2020.